apps/passport-server/src/services/generic-issuance/pipelines/LemonadePipeline.ts

```diff
@@ -802,14 +808,7 @@ export class LemonadePipeline implements BasePipeline {
         }
       }
 
-      const email = emailClaim.emailAddress.toLowerCase();
```

Comment:
I don't think we replicate this `toLowerCase` call in the new code path, and it _might_ be important.

Reply:
I think I do it here - https://github.com/proofcarryingdata/zupass/blob/ivan/faster-ticket-loading/apps/passport-server/src/services/generic-issuance/subservices/CredentialSubservice.ts#L57